www/php/models/Reports/Sample.php

```diff
@@ -54,4 +55,4 @@ public function getReportData($value=''){
     }
 }
 
-?>
\ No newline at end of file
+?>
```

Comment:
closing `?>` is not required in php scripts

Reply:
github shows the new line added diff. it's a github online editor that adds these new lines.